MAUI/Cartesian-Charts/Annotation.md

```diff
@@ -0,0 +1,743 @@
+---
+layout: post
+title: Annotation in .NET MAUI Chart control | Syncfusion
+description: Learn here all about annotation support and its features in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Annotation in .NET MAUI Chart
+
+[SfCartesianChart]() supports annotations that allow you to mark specific areas of interest in the chart area. You can add text, images, and custom views.
+
+The following annotations are supported in [SfCartesianChart]():
+
+* Text annotation
+* Shape annotation
+* View annotation
+
+## Adding Annotations
+
+You can create an instance of any type of annotation and add it to the Annotations collection. For example, the [EllipseAnnotation]() can be added.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCartesianChart>
+    ...
+        <chart:SfCartesianChart.Annotations>
+
+        <chart:EllipseAnnotation X1="2" X2="4" Y1="10" Y2="15" Text="Ellipse"/>
+        
+        </chart:SfCartesianChart.Annotations>
+    
+    </chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+    ...
+    var ellipse = new EllipseAnnotation()
+    {
+        X1 = 2,
+        Y1 = 10,
+        X2 = 4,
+        Y2 = 15,
+        Text = "Ellipse"
+    };
+
+    chart.Annotations.Add(ellipse);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Annotation in MAUI Chart](Annotation_image/ellipse_annotation.png)
+
+## Positioning the annotation
+
+[Annotations]() can be positioned in the plot area based on the [X1]() and [Y1]() properties. For shape annotations, specify the [X2]() and [Y2]() properties, if needed. The X and Y values can be specified using axis units or pixel units, which can be identified by the [CoordinateUnit]() property.
+
+## Positioning based on CoordinateUnit as axis
```

Comment:
Why do we need to mention separate categories for pixel and axis range base? Update the code snippet as we discussed. 

Reply:
@SaiyathAliFathima , removed the axis based code and content